tests/core/full_node/test_mempool.py

```diff
@@ -533,7 +628,7 @@ async def test_correct_announcement_empty_message(self, two_nodes):
         assert mempool_bundle is bundle
 
     @pytest.mark.asyncio
-    async def test_invalid_announcement_consumed(self, two_nodes):
+    async def test_invalid_puzzle_announcement_consumed(self, two_nodes):
```

Comment:
Maybe instead of `consumed` use `rejected` (since we're testing a rejection here).